code/__DEFINES/events.dm

```diff
@@ -7,3 +7,15 @@
 #define EVENT_READY 1
 #define EVENT_CANCELLED 2
 #define EVENT_INTERRUPTED 3
+
+#define EVENT_CAT_AI "AI issues"
+#define EVENT_CAT_ANOMALIES "Anomalies"
+#define EVENT_CAT_BUREAUCRATIC "Bureaucratic"
+#define EVENT_CAT_ENGINEERING "Engineering"
+#define EVENT_CAT_ENTITIES "Entities"
+#define EVENT_CAT_FRIENDLY "Friendly"
+#define EVENT_CAT_HEALTH "Health"
+#define EVENT_CAT_HOLIDAY "Holiday"
+#define EVENT_CAT_INVASION "Invasion"
+#define EVENT_CAT_SPACE "Space Threats"
+#define EVENT_CAT_WIZARD "Wizard"
```

Comment:
1. Doc-comment all these (///)
2. CATEGORY, not CAT

Reply:
All done.